2.6.0/spec/services.md

```diff
@@ -0,0 +1,269 @@
+# Services Endpoint
+
+Used to fetch a list of all available services from a CO belonging to the current SP.
+
+## Methods
+
+### GET
+
+#### Available services for a single Access
+
+To get available services for a single access use accessId as key. The response contains details about activated services and the services
+ available for activation.
+ 
+Request
+```http
+GET /onapi/2.6/services/{accessId} HTTP/1.1
```

Comment:
I would prefer
```http
GET /onapi/2.6/services/?accessId={accessId} HTTP/1.1
```
As we are fetching services not accesses the in path id should be reserved for a `serviceId` (which we don't have but to keep the formatting consistent we shouldn't use it for something else)

Reply:
See https://github.com/on-api/on-api/pull/88#discussion_r1294591413 for more discussions regarding all three endpoints.